gempython/tools/optohybrid_user_functions_xhal.py

```diff
@@ -483,12 +483,12 @@ def performDacScan(self, outData, dacSelect, dacStep=1, mask=0x0, useExtRefADC=F
             exit(os.EX_USAGE)
 
         #Check length of results container
-        lenExpected = (maxVfat3DACSize[dacSelect][0] - 0+1)*24 / dacStep
+        lenExpected = (maxVfat3DACSize[dacSelect][0] - 0+1)*self.parentAMC.nVFAT / dacStep
```

Comment:
this will actually cause an `AttributeError` no? e.g. [same](https://github.com/cms-gem-daq-project/cmsgemos/pull/287/files#r297225193)